pkg/webui/lib/components/date-time/index.js

```diff
@@ -63,7 +63,7 @@ class DateTime extends React.PureComponent {
     }
 
     return (
-      <time className={className} dateTime={dateValue.toISOString()} title={result}>
+      <time className={className} dateTime={dateValue.toISOString()}>
```

Comment:
We don't want to remove the title entirely from the component. It should rather be controlled by a prop e.g. `noTitle`.